build/loaders/convert-md-to-doc.js

```diff
@@ -86,13 +86,15 @@ function genScript (demoInfos, components = [], url, forceShowAnchor) {
     .map(({ variable, fileName }) => `import ${variable} from './${fileName}'`)
     .concat(
       components.map(
-        (component) => `import ${camelCase(component)} from './${component}'`
+        ({importStmt}) => importStmt
```

Comment:
感觉 pre-commit hook 配置的不正确，研究一下吧。

```suggestion
        ({ importStmt }) => importStmt
```